app/models/fiber_channel_address.rb

```diff
@@ -0,0 +1,7 @@
+class FiberChannelAddress < SanAddress
+  def self.class_by_ems(ext_management_system)
+    # TODO(lsmola) taken from Orchestration stacks, correct approach should be to have a factory on ExtManagementSystem
+    # side, that would return correct class for each provider
+    ext_management_system && ext_management_system.class::FiberChannelAddress
+  end
```

Comment:
I think we don't need class_by_ems, since we don't expect EMSs do have their own Address classes. 
@agrare , what do you think?

Reply:
:+1: Correct

Reply:
done